libs/components/src/lib/video-player/video-player.spec.ts

```diff
@@ -0,0 +1,286 @@
+			expect(element.loop).toBe(false);
+			expect(element.skipBy).toBe(MediaSkipBy.Ten);
+			expect(element.playbackRates).toBe(DEFAULT_PLAYBACK_RATES);
+		});
+
+		it('should show the big play button by removing the vjs-hidden class', async () => {
+			const bigPlayBtn = getBigPlayButton();
+			expect(bigPlayBtn?.classList.contains('vjs-hidden')).toBe(false);
+		});
+
+		it('should remove the redundant lang attribute', () => {
+			expect(element.shadowRoot?.querySelector('[lang]')).toBe(null);
+		});
+	});
+
+	describe('src', () => {
+		it('should show the big play button by removing the vjs-hidden class', async () => {
+			element = (await fixture(
+				`<${COMPONENT_TAG} src="${VIDEO_SRC}"></${COMPONENT_TAG}>`
+			)) as VideoPlayer;
+			await elementUpdated(element);
+			const bigPlayBtn = getBigPlayButton();
+			expect(bigPlayBtn?.classList.contains('vjs-hidden')).toBe(false);
+		});
+
+		it('should allow the src to be updated', async () => {
+			element = (await fixture(
+				`<${COMPONENT_TAG} autoplay>
+					<source src="${VIDEO_SRC}" type="video/mp4">
+				</${COMPONENT_TAG}>`
+			)) as VideoPlayer;
+			await elementUpdated(element);
+			element.src = 'new-src.mp4';
+			await elementUpdated(element);
+			const bigPlayBtn = getBigPlayButton();
+			expect(bigPlayBtn?.classList.contains('vjs-hidden')).toBe(false);
+		});
+	});
+
+	describe('autoplay', () => {
+		it('should set the muted attribute on the video element', async () => {
+			element = (await fixture(
+				`<${COMPONENT_TAG} autoplay>
+					<source src="${VIDEO_SRC}" type="video/mp4">
+				</${COMPONENT_TAG}>`
+			)) as VideoPlayer;
+			const videoEle = getVideoEle();
+			expect(videoEle?.autoplay).toBe(true);
+		});
+	});
+
+	describe('loop', () => {
+		it('should set the loop attribute on the video element', async () => {
+			element = (await fixture(
+				`<${COMPONENT_TAG} loop>
+					<source src="${VIDEO_SRC}" type="video/mp4">
+				</${COMPONENT_TAG}>`
+			)) as VideoPlayer;
+			const videoEle = getVideoEle();
+			expect(videoEle?.loop).toBe(true);
+		});
+	});
+
+	describe('playback rates', () => {
+		it('should set custom playback rates', async () => {
+			element = (await fixture(
+				`<${COMPONENT_TAG} playback-rates="0.25, 0.5, 1">
+					<source src="${VIDEO_SRC}" type="video/mp4">
+				</${COMPONENT_TAG}>`
+			)) as VideoPlayer;
+			await elementUpdated(element);
+			const playbackRatesMenuItems = element.shadowRoot?.querySelectorAll('.vjs-playback-rate .vjs-menu li');
+			expect(playbackRatesMenuItems?.length).toBe(3);
+			if (playbackRatesMenuItems?.length === 3) {
+				expect(playbackRatesMenuItems[0].querySelector('.vjs-menu-item-text')!.textContent).toBe('1x');
+				expect(playbackRatesMenuItems[1].querySelector('.vjs-menu-item-text')!.textContent).toBe('0.5x');
+				expect(playbackRatesMenuItems[2].querySelector('.vjs-menu-item-text')!.textContent).toBe('0.25x');
+			}
+		});
+
+		it('should disable playback rates when passed an empty string', async () => {
```

Comment:
According to the test, it should hide the playback rates, right?
```suggestion
		it('should hide playback rates when passed an empty string', async () => {
```